lib/compat/wordpress-5.9/block-template-utils.php

```diff
@@ -474,12 +472,8 @@ function _remove_theme_attribute_in_block_template_content( $template_content )
 			}
 		}
 
-		if ( $has_updated_content ) {
-			foreach ( $template_blocks as $block ) {
-				$new_content .= serialize_block( $block );
-			}
-
-			return $new_content;
+		if ( ! $has_updated_content ) {
+			return $template_content;
 		}
 
 		return $template_content;
```

Comment:
This `if` is now unnecessary. In both cases we `return $template_content;`. Is that right?